SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -1,114 +1,28 @@
-      then a file called "ManagedFileList" will be written containing the list of managed code
-      files that was passed to the compiler.
-      
-      The output will only be produced if the properties $(SonarQubeOutputPath) and 
-      $(RunSonarQubeAnalysis) are specified. $(SonarQubeOutputPath) is the output directory 
-      in which the output file should be dropped and can be relative or absolute 
-      (use an absolute path if you want a consolidated set of output for a single build).
-
-      Using this targets file
-      - - - - - - - - - - - -
-      To use these targets with a single project, import the targets into that project.
-
-      To use these targets with all projects, put the file in:
-          C:\Program Files (x86)\MSBuild\12.0\Microsoft.Common.Targets\ImportBefore
-      ... adjusting the MSBuild version as necessary.
-
-
-      Example of use:
-      - - - - - - - -
-      msbuild MyProject.csproj /p:SonarQubeOutputPath=C:\SQOutput /p:RunSonarQubeAnalysis=true
-
-
-      Excluding projects and files from SonarQube analysis
-      ****************************************************
-      Individual projects can excluded from SonarQube analysis by setting
-      the property $(SonarQubeExclude) e.g.
-      
-        <SonarQubeExclude>true</SonarQubeExclude>
-
-      Individual files can be excluded from analysis by setting the 
-      <SonarQubeExclude> metadata item e.g.
-      
-        <Compile Include="Logger.cs">
-          <SonarQubeExclude>true</SonarQubeExclude>
-        </Compile>
-  
-  
-      Test projects
-      *************
-      The analysis performed by SonarQube varies depending on whether a project
-      is a test project or not.
-
-      A project can be explicitly marked as being a test or product project by setting
-      the property $(c) e.g.
-      
-        <SonarQubeTestProject>true</SonarQubeTestProject>
-       
-      If the $(SonarQubeTestProject) is not set then the targets will determine the project
-      type as follows:
-      
-      1) Projects whose full project file name matches the regular expression in the analysis
-         config file will be treated as test projects. This is set via the SonarQube portal
-         (the "Test project pattern" in the per-project settings for C#).
-          By default, any projects with:
-          * "test" (case-insensitive) in the project name, or
-          * that have a directory called "test" or "tests" in the path
-          will be treated as test projects
-          
-          Note that the regular expression uses the .Net Regex format i.e. "+" is the
-          single character wildcard and "*" is the multi-character wildcard (zero to many).
-          
-      2) MSTest projects will be treated as test projects.
-          The $(ProjectTypeGuid) of MS Test projects contains a specific guid
-          ("3AC096D0-A1C2-E12C-1390-A8335801FDAB")
```

Comment:
We've found in practice that it's generally helpful to end users to include some documentation in a targets file. Users often have customised builds that require them to make some tweaks to get their custom builds working as they want. Providing some documentation in the targets file helps with this and reduces the number of support questions.